lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -72,6 +73,18 @@ sub _respond_error_for_group_overview ($self, $error) {
     );
 }
 
+sub _sort_info_time ($new_old) { "Builds are sorted by the creation time of the <em>$new_old</em> job." }
+
+sub _sort_help_timestamps ($new_old) {
+"This means the timestamps are the creation time of the <em>$new_old</em> job in that build (accross all architectures).";
```

Comment:
Are there any reasons for 0 indentation?

```suggestion
    "This means the timestamps are the creation time of the <em>$new_old</em> job in that build (accross all architectures).";
```

Reply:
Yes, tidy prefers it this way.